### PR TITLE
renames arquillian pact to arquillian algeron

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

File: _ext/arquillian.rb [hunk 1]
```diff
@@ -467,7 +467,7 @@ def visit(repository, site)
           'org_jboss_arquillian_core' => 'Arquillian Core',
           'arquillian_cube' => 'Arquillian Cube Extension',
           'arquillian_cube_q' => 'Arquillian Cube Q Extension',
-          'arquillian_pact' => 'Arquillian Pact Extension',
+          'algeron' => 'Arquillian Algeron Extension',
```
Comment (bartoszmajsak) [inline _ext/arquillian.rb]: Did you run the build? Shouldn't it be `arquillian_algeron`?

Comment (lordofthejars) [inline _ext/arquillian.rb]: Probably you are right. The problem is that I cannot run the build. It always failing

**Comment by bartoszmajsak** [comment]
I will test it.

**Comment by bartoszmajsak** [comment]
This is not as trivial as just rename. We have to provide following things extra

- [ ] Rename old blogs to match algeron
- [ ] Add a note in old blogs that the project has been renamed (hence different title), but artifacts are available under old name
- [ ] Take care of redirects for the old links (as they went out into the wild)

I will take care of it.

**Comment by bartoszmajsak** [comment]
Cherry-picked to develop.

**Pull request closed without merging**